runtime/stdlib/contracts/test.cdc

```diff
@@ -12,6 +12,103 @@ pub contract Test {
             self.backend = backend
         }
 
+        /// Returns the Flow token balance for the given account.
+        ///
+        pub fun getFlowBalance(_ account: Test.Account): UFix64 {
+            let script = "pub fun main(address: Address): UFix64 { return getAccount(address).balance }"
+            let scriptResult = self.executeScript(script, [account.address])
+
+            if scriptResult.status == ResultStatus.failed {
+                panic(scriptResult.error!.message)
+            }
+            return scriptResult.returnValue! as! UFix64
+        }
+
+        /// Returns the current block height of the blockchain.
+        ///
+        pub fun getCurrentBlockHeight(): UInt64 {
+            let script = "pub fun main(): UInt64 { return getCurrentBlock().height }"
+            let scriptResult = self.executeScript(script, [])
+
+            if scriptResult.status == ResultStatus.failed {
+                panic(scriptResult.error!.message)
+            }
+            return scriptResult.returnValue! as! UInt64
+        }
+
+        /// Mints the given amount of Flow tokens to a specified test account.
+        /// The transaction is authorized and signed by the service account.
+        /// Returns the result of the transaction.
+        ///
+        pub fun mintFlow(_ receiver: Test.Account, amount: UFix64): TransactionResult {
+            let lines: [String] = [
```

Comment:
I would have liked to be able to write this in a multi-line `String`, but that isn't possible yet.
So I had to improvise, in order to make this somewhat more readable :innocent: